frontend/src/app/components/dashboard/db-table-view/saved-filters-panel/saved-filters-dialog/saved-filters-dialog.component.css

```
@@ -628,6 +651,295 @@
+}
+
+.filters-header__table {
+	font-size: 12px;
+	font-weight: 500;
+	opacity: 0.6;
+}
+
+@media (width <= 600px) {
+	.filters-content {
+		grid-template-columns: 32px 1fr auto !important;
+		grid-row-gap: 0 !important;
```

Stylelint (17.12.0)

[error] Expected "grid-row-gap" to be "row-gap" (property-no-deprecated)

(property-no-deprecated)